### ISSUE TITLE
An error occurred attempting to determine the process id of dotnet.exe

### ISSUE THREAD

**Issue by UpperAvenue** [aspnet/Home#1723]
When attempting to run my application with SSL enabled, I get the following error:

> An error occurred attempting to determine the process id of dotnet.exe which is hosting your application.  One or more errors occured.

It seems to happen when the App URL begins with 'https://' regardless of whether Enable SSL has been selected.  What would be causing this error?

**Comment by Tratcher** [comment]
How did you configure it?

**Comment by UpperAvenue** [comment]
What do you mean?

**Comment by davidfowl** [comment]
Can you put your application(GitHub preferable) somewhere where we can reproduce the problem.

**Comment by UpperAvenue** [comment]
Here it is.  https://github.com/UpperAvenue/HttpsConfiguration

**Comment by thedole** [comment]
I came here searching for help on this same issue. The fix for me was found in Vackups answer to [this Stack Overflow question](http://stackoverflow.com/questions/34197265/an-error-occurred-attempting-to-determine-the-process-id-of-the-dnx-process-host
)

In short, the fix for me was this:
"So, running the site with Ctl+F5 installed the certificate and then I was able to debug the site solving my issue."

**Comment by UpperAvenue** [comment]
That did it - thank you!

**Comment by jesselton23** [comment]
Thank you. I was going crazy why this error comes up after enabling ssl. 👍

**Comment by pulverize** [comment]
I keep circling around to this issue. Ctrl-F5 is not fixing it for me.

**Comment by Doff3n** [comment]
I did [this:](https://stackoverflow.com/a/46926381/1304590)

I had been messing with some certificates, so I had removed the IIS Express certificate. So easiest way to reinstall the certificate I found before coming here was to go into add/remove programs and run fix IIS Express, which installs the certificate again. Not sure if Ctrl+f5 does the same thing.

After that I had to disable and re-enable ssl. 

This does not really feel like a developer solution, more trying and failing, but it worked!
